Polish docs for the v1.1.0 release (#221)
harvester/docs: docs/index.md

| Type | Requirements                                                                                                                                                                                               |
|:---|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above preferred for production                                                                |
| Memory | 32 GB minimum; 64 GB or above preferred                                                                                                                                                                    |

> | Disk Capacity | 140 GB minimum for testing; 500 GB or above preferred for production                                                                                                                                       |

Reviewer: 200GB for testing upgrade.

Author: Sure will update the minimum to 200GB

3. Choose the installation device on which the Harvester cluster will be installed
    - Note: By default, Harvester uses [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) partitioning schema for both UEFI and BIOS. If you use the BIOS boot, then you will have the option to select [MBR](https://en.wikipedia.org/wiki/Master_boot_record).
   ![iso-install-disk.png](/img/v1.1/install/iso-install-disk.png)

> 5. Recommended choosing a separate disk for storing VM data.

Reviewer 2: ```suggestion
5. We recommend choosing a separate disk for storing VM data.
```

   ![iso-install-disk.png](/img/v1.1/install/iso-select-data-disk.png )

> 6. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.

Reviewer 2: ```suggestion
6. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
```

   ![iso-installed.png](/img/v1.1/install/iso-nic-config.png)
7. Optional: Configure the DNS servers; use commas as delimiters.

> 8. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP, VIP must be different than any Node IP)</small>.

Author: This changes the original meaning, so I'll skip this one.

9. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
10. Configure the login password of the host. The default SSH user is `rancher`.

> 11. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.

Reviewer 2: ```suggestion
11. We recommend configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.
```

12. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here. Otherwise, leave this blank.
13. (Optional) You can choose to import SSH keys from a remote server URL. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
14. (Optional) If you need to customize the host with a [Harvester configuration](./install/harvester-configuration.md) file, enter the HTTP URL here.

> 15. Confirm the installation options and Harvester will be installed to your host. The installation may take a few minutes to complete.

Reviewer 2: ```suggestion
15. Confirm the installation options and Harvester will be installed on your host. The installation may take a few minutes to complete.
```
